### DISCUSSION TITLE
Support for previousState

### DISCUSSION THREAD

**Discussion by wollyd** [seaside1/jrule#163] [Q&A]
Hi,

is it possible to add the previousState of an item (see org.openhab.core.persistence.extensions.PersistenceExtensions).
That would be very nice 

Thanks

Willy

**Comment by querdenker2k** [comment]
see https://github.com/seaside1/jrule/pull/168

**Reply by wollyd** [reply]
ok thanks a lot

**Comment by seaside1** [comment]
This has been merged now-
